cmd/harmony/config/default.go

```diff
@@ -0,0 +1,213 @@
+package config
+
+import nodeconfig "github.com/harmony-one/harmony/internal/configs/node"
+
+const TOMLConfigVersion = "2.0.0"
+
+const (
+	DefNetworkType = nodeconfig.Mainnet
```

Comment:
Make items in this file private?

Reply:
They are used in flags_test.go unfortunately. another option that I considered is to move flags-related code into config package as well, but I declined the idea, since flags are not 100% following the config.

Reply:
Ah I see. Please just keep it as it is, thanks